tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
```

Comment:
Might be best to rename the output directory to `dist` if it contains both `src` and `tools`. Though we probably don't even need to transpile `tools` if we add typescript support to `react-benchmark` (should be pretty easy to do).